GooglePlayPlugins/com.google.play.billing/Proguard/billing.txt

```diff
@@ -8,7 +8,9 @@
 
 # Keep the AIDL interface
 -keep class com.android.vending.billing.**
+-keepclassmembers class com.android.vending.billing.** { *; }
```

Comment:
Remove this line and just add { *; } to previous line, does that work? Same comment for the other place.

Reply:
> Remove this line and just add { *; } to previous line, does that work? Same comment for the other place.

I tried and it works!